crates/jstz_cli/build.rs

```diff
@@ -0,0 +1,62 @@
+use std::{
```

Comment:
Can you please provide some more description of what is the expected behaviour of running this?